wagtail/contrib/modeladmin/tests/test_page_modeladmin.py

```diff
@@ -157,6 +157,10 @@ def test_non_existent(self):
         response = self.get(100)
         self.assertEqual(response.status_code, 404)
 
+    def test_label_display(self):
+        response = self.client.get('/admin/modeladmintest/author/inspect/1/')
```

Comment:
Could you add a short comment to this test to explain briefly explain what you're expecting to see and why? This should make maintenance a little easier.

Reply:
OR rename the test to something more explanatory, e.g. `test_short_description_is_used_for_author_birth_string_label()`, and maybe also test that "author_birth_string" is NOT present?